cfme/tests/configure/test_config_manual.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream("5.9", "5.10", "upstream")]
```

Comment:
Need to modify this to collect tests against the versions that the test is relevant for.